PROJECTS.md

```diff
@@ -0,0 +1,48 @@
+# OpenJS Foundation Projects
+
+The projects below are full OpenJS Foundation members at the stage level indicated in the heading. Projects may move to new stages upon annual review or by request. 
```

Comment:
Optional simplification: Split long-ish sentence into two short and direct sentences:

```suggestion
The projects below are full OpenJS Foundation members. They are at the stage level indicated in the headings. Projects may move to new stages upon annual review or by request. 
```